src/core.js

```diff
@@ -1527,6 +1527,8 @@ Strophe.Connection = function (service, options)
     this.removeHandlers = [];
     this.addTimeds = [];
     this.addHandlers = [];
+    //HttpError handlers
+    this.httpErrorHandler = {};
```

Comment:
This object contains potentially multiple handlers, so please name it `httpErrorHandlers` (like the ones above it).